src/catch2/catch_user_config.hpp.in

```diff
@@ -130,6 +130,16 @@
 
 
 
+#cmakedefine CATCH_CONFIG_GETENV
+#cmakedefine CATCH_CONFIG_NO_GETENV
```

Comment:
It also needs to be added to `CMake/CatchConfigOptions.cmake`, so that the option is shown for people using CMake frontends.